src/dotnet/APIView/APIViewWeb/Languages/PythonLanguageService.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using ApiView;
+
+namespace APIViewWeb
+{
+    public class PythonLanguageService : ILanguageService
```

Comment:
Seems like there is a lot of duplication with Java service, can we extract a base class with common functionality please?

Reply:
@pakrym  I have moved common code into an abstract base class that is inherited by java and python language service classes.